packages/core-transaction-pool-mem/lib/connection.js

```diff
@@ -38,16 +38,24 @@ class TransactionPool extends TransactionPoolInterface {
 
     this.__purgeExpired()
 
-    // Remove transactions that were forged while we were offline.
-    const allIds = all.map(
```

Comment:
Why move to a separate function? It is called in just one place. This patch removes 4 lines and adds 41!